app/controllers/project_controller.rb

```diff
@@ -3,47 +3,38 @@ class ProjectController < ApplicationController
   before_action :authenticate_newuser!
 
   def index
-    @project = Project.where("newuser_id = ?", current_newuser.id.to_s)
+    # @project = Project.where("newuser_id = ?", current_newuser.id.to_s)
+    @projects = current_newuser.projects
+    # byebug
```

Comment:
Remove byebug comments
Only use byebug for debugging purposes and never commit it to github